src/app/core/services/user.service.ts

```diff
@@ -302,8 +302,8 @@ export class UserService extends CommonService {
           tokenId: payload.tokenId,
           actionIn: payload.isNFTDetail ? null : !payload.isTransferTab ? CW721_TRACKING : null,
           actionNotIn: payload.isNFTDetail ? null : payload.isTransferTab ? ['approve', 'instantiate', 'revoke'] : null,
-          neqCw4973: payload.isCW4973 ? null : 'crates.io:cw4973',
-          eqCw4973: payload.isCW4973 ? 'crates.io:cw4973' : null,
+          neqCw4973: 'crates.io:cw4973',
```

Comment:
chỗ này bỏ luôn đi được không em?

Reply:
Nếu bỏ ra thì mấy cái CW4973 sẽ trả về luôn nha anh

Reply:
à not equal ở câu query ok em.